src/UniversalRewardsDistributor.sol

```diff
@@ -83,7 +83,7 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributorStaticTyping
         if (timelock == 0) {
             _setRoot(newRoot, newIpfsHash);
         } else {
-            pendingRoot = PendingRoot(block.timestamp, newRoot, newIpfsHash);
+            pendingRoot = PendingRoot(block.timestamp + timelock, newRoot, newIpfsHash);
             emit EventsLib.RootProposed(newRoot, newIpfsHash);
```

Comment:
would it make sense to also log when the pending proposal timelock will expire?

Reply:
There's no expiration on the URD, maybe you mixed it with MM?

Reply:
sorry, I meant when it would be "acceptable".

Reply:
In this case it's already the case since we log `block.timestamp + timelock` no?

Reply:
Where is such a log? The current event does not log such information.

Reply:
nevermind I mistake the code lines. @julien-devatom do you think it's necessary?